### DISCUSSION TITLE
Speeding up simulation execution

### DISCUSSION THREAD

**Discussion by zhekai-jiang** [skhatiri/UAV-Testing-Competition#15] [Platform]
We notice that the simulation for the given sample missions usually takes over 3 minutes to run. Is there any way we can speed it up so that we can test our generator more efficiently?

I have tried to adjust the `speed` parameter in the missions' yaml file and to set it when initializing the generator object in the Python code (via `drone_test.test.speed` and `drone_test.simulation.speed`). But none of them seems to work. Any advice is appreciated!

**Comment by dgumenyuk** [comment]
@zhekai-jiang Thank you for your question! The feature to speed up the simulation is currently under development and not yet stable. Because of this, we recommend running simulations at the default speed. For now, we suggest keeping the "speed" parameter in the .yaml file set to the default value of 1, which will also be used for evaluation.
